src/action.py

```diff
@@ -56,21 +72,28 @@ def _create(
 
     Returns:
         A report on the outcome of executing the action.
+
+    Raises:
+        NotImplementedError: if a requested action has not been implemented yet.
```

Comment:
Since the code is not expecting the error to be ever raised (line 107), should this be in the docstring?

Reply:
It gets enforced by our tooling - it also seems good to at least document the possibility that it might be raised